source/globalCommands.py

```diff
@@ -1159,7 +1164,15 @@ def script_navigatorObject_current(self, gesture: inputCore.InputGesture):
 				else:
 					api.copyToClip(text, notify=True)
 		else:
-			speech.speakObject(curObject, reason=controlTypes.OutputReason.QUERY)
+			speechList = speech.getObjectSpeech(curObject, reason=controlTypes.OutputReason.QUERY)
+			speech.speech.speak(speechList)
+			for i in copy.copy(speechList):
+				if not isinstance(i, str):
+					speechList.remove(i)
+			text = ' '.join(speechList)

```

Comment:
I think this would be more standard logic - we also don't need to import copy

```suggestion
			brailleList = speechList.copy()
			for i in speechList:
				if not isinstance(i, str):
					brailleList.remove(i)
			text = ' '.join(brailleList)
```